frontend/integration-tests/tests/deploy-image.scenario.ts

```diff
@@ -12,15 +12,26 @@ describe('Deploy Image', () => {
   const imageName = 'mysql';
 
   describe('Deploy Image page', () => {
-    it('should auto-fill the namespace/project', async() => {
+    it('should render project/namespace dropdown when all-namespace is selected', async() => {
       // Navigate to the deploy-image page
-      await browser.get(`${appHost}/deploy-image/ns/${testName}?preselected-ns=${testName}`);
+      await browser.get(`${appHost}/deploy-image/all-namespaces`);
```

Comment:
I do NOT think it is even possible to navigate to this URL. I get a 404 when I try it. 
From the main page, if I select `all-namespaces` I can NOT go to the `Add` page, it redirects me to the "select a project" page.